gensrc/script/gen_build_version.sh

```diff
@@ -27,7 +27,13 @@
 
 set -eo pipefail
 
-build_version="trunk"
+build_version_prefix="doris"
```

Comment:
Does it need to be manually modified when we publish the version?

Reply:
yes, it needs modify version info manually in this commit

Reply:
Can it be specified at compile?

Reply:
you can specify any version info you want in this script when you complie